src/components/search-group/scrollable-groups-result/ScrollableGroupsResult.jsx

```diff
@@ -71,29 +76,36 @@ const ScrollableGroupsResult = ({
   );
 
   const renderGroupsList = () => {
-    if (((privateGroups && privateGroups.length === 0)
-      && (publicGroups && publicGroups.length === 0))) {
+    if (privateGroups?.length === 0 && publicGroups?.length === 0) {
       return noGroupsFound;
     }
-    if (currentUserGroups && currentUserGroups.length === 0) {
+
+    if (currentUserGroups?.length === 0) {
       return noCurrentUserGroupsFound;
     }
+
     return (
       <Fade in>
         <div className={classes.wrap}>
-          {currentUserGroups && currentUserGroups.length > 0 && (currentUserGroupsList)}
-          {privateGroups && privateGroups.length > 0 && (privateGroupsList)}
-          {publicGroups && publicGroups.length > 0 && (publicGroupsList)}
+          {currentUserGroups?.length && currentUserGroupsList}
+          {privateGroups?.length && privateGroupsList}
+          {publicGroups?.length && publicGroupsList}
```

Comment:
```suggestion
          {currentUserGroups?.length > 0 && currentUserGroupsList}
          {privateGroups?.length > 0 && privateGroupsList}
          {publicGroups?.length > 0 && publicGroupsList}
```

If you search only `מבצע סודי` for example, you will get `0` on public groups (and it shows it)